package.json

```diff
@@ -95,5 +94,8 @@
       "sass-loader",
       "webpack"
     ]
+  },
+  "dependencies": {
```

Comment:
common practice to keep `dependencies` above `devDependencies`.

Reply:
That's strange because it was done automatically by `npm`

Reply:
Actually, I don't know how exactly `npm` sorting deps and devDeps. Maybe it depends on which one was added first. But it will make more sense if `npm` has a strict order for them. In this case better to follow that order.

Reply:
Tested it now, seems like `npm` just adding new section into the end of file. So I will move deps to position before devDeps.